src/Polly/CircuitBreaker/ICircuitBreakerPolicy.cs

```diff
@@ -31,7 +31,7 @@ public interface ICircuitBreakerPolicy : IsPolicy
 /// Defines properties and methods common to all circuit-breaker policies generic-typed for executions returning results of type <typeparamref name="TResult"/>.
 /// </summary>
 /// <typeparam name="TResult">The type of the result.</typeparam>
-public interface ICircuitBreakerPolicy<TResult> : ICircuitBreakerPolicy
+public interface ICircuitBreakerPolicy<out TResult> : ICircuitBreakerPolicy
```

Comment:
I'm going to have to read up on whether these changes are considered breaking or not.

Reply:
The third reply under this Stack Overflow answer by Eric Lippert seems to suggest it's a breaking change: https://stackoverflow.com/a/3603459